src/scribe-plugin-underliner.js

```diff
@@ -0,0 +1,41 @@
+module.exports = function(config) {
+    "use strict";
+
+
+  return function(scribe) {
+      const template = (cssClass, term) => {
+          return ['<span class="', cssClass, '">', term, '</span>'].join('');
+      };
+
+      const {cssClass} = config;
+      const underlinerCommand = new scribe.api.Command('underliner');
+      underlinerCommand.queryEnabled = () => { return true; };
+
+
+      underlinerCommand.execute = (terms) => {
```

Comment:
Suggestion: for "public" functions such as this, I prefer to use named functions instead of anonymous as the name will show in stack traces on errors, which is very handy for debugging. So the above can be written as:

```js
underlinerCommand.execute = function underlinerCommandExecute( terms ) {
   // stuff
};
```